client/storybook/Welcome.stories.mdx

```diff
@@ -0,0 +1,7 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+
+<Meta title="Welcome" />
+
+# Welcome to Wagtail’s design system
+
+Wagtail’s admin interface is built with reusable UI components, structured as a design system for ease of maintenance and third-party reuse.
```

Comment:
This is just a placeholder, actual content for this "home" screen will be further fleshed out once we have more docs in here (design tokens, an overview of all components, npm package, etc.).